<launch>

	<node pkg="rosserial_arduino" name="serial_node" type="serial_node.py" output="screen" respawn="true">
	<param name="port" type="string" value="/dev/ttyACM0"/>
	<param name="baud" type="int" value="1000000"/>
	</node>
	
	<node pkg="dynamixel" name="panTiltAngle" type="panTiltAngle.py" output="screen"/>
	<node pkg="dynamixel" name="convertStatus" type="convertStatus.py" output="screen"/>

</launch>